tests/dragonfly/utility.py

```diff
@@ -141,23 +143,35 @@ def rand_str(k=3, s=''):
 
         if t == ValueType.STRING:
             # Random string for MSET
-            return rand_str(self.val_size)
+            return (rand_str(self.val_size),)
         elif t == ValueType.LIST:
             # Random sequence k-letter elements for LPUSH
-            return ' '.join(rand_str() for _ in range(self.val_size//4))
+            return tuple(rand_str() for _ in range(self.val_size//4))
```

Comment:
I had issues with correctly escaping the string values when formatting cmds, so I switched to tuples (which is more correct)